site/webpack.config.js

```diff
@@ -0,0 +1,48 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+// directory where compiled assets will be stored
+    .setOutputPath('web/build')
+    // public path used by the web server to access the output path
+    .setPublicPath('/build')
+    // only needed for CDN's or sub-directory deploy
+    //.setManifestKeyPrefix('build/')
+
+    /*
+     * ENTRY CONFIG
+     *
+     * Add 1 entry for each "page" of your app
+     * (including one that's included on every page - e.g. "app")
+     *
+     * Each entry will result in one JavaScript file (e.g. app.js)
+     * and one CSS file (e.g. app.css) if you JavaScript imports CSS.
+     */
+    .addEntry('app', ['./web/assets/scss/librecores.scss',
```

Comment:
Instead of adding all files here, create an `app.scss` and include CSS files inside it.

Reply:
That's a good idea. Thanks!